### PR TITLE
Added login view and URL patterns

### PR DESCRIPTION
**Pull request by clintonb**
We are duplicating this code across our repositories. This codifies those duplications into a single location.

ECOM-4607

### CODE REVIEW

**Comment by clintonb** [comment]
FYI @edx/ecommerce

File: auth_backends/urls.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+""" Shared URL patterns.
+
+Add these to your project's `urlpatterns` to avoid duplicating code.
+"""
+from django.conf.urls import url, include
+
+from auth_backends.views import EdxOpenIdConnectLoginView
+
+auth_urlpatterns = [  # pylint: disable=invalid-name
+    url(r'^login/$', EdxOpenIdConnectLoginView.as_view(), name='login'),
```
Comment (rlucioni) [inline auth_backends/urls.py]: I take it that someone wanting to use this is going to have to include their subclassed version of `LogoutRedirectBaseView` here? Would a comment to that effect make sense here?

Comment (clintonb) [inline auth_backends/urls.py]: README updated.

**Comment by rlucioni** [comment]
👍

**Pull request merged**